src/xdocs/report_issue.xml

```diff
@@ -174,8 +208,42 @@ expected level should be 12. [Indentation]
         The more examples the better.
       </p>
       <p>
-        If you have open source project, please send a link to the whole java file.
+        Explain the rationale behind implementing the new check with relevant
+        references (if possible).
       </p>
+      <p>
+        Feel free to propose a name for the new Check, along with all its attributes.
+      </p>
+      <p>
+        If you have open source project, please give a link to the whole java file.
+      </p>
+      <p>
+        Example of Check/Module Request that we expect:
+      </p>
+      <source>
+Describe the problem in detail:
+
+/var/tmp $ cat config.xml
+<module name="YourNewModule">
+  <property name="yourPossibleProperty1" value="true"/>
+  <property name="yourPossibleProperty1" value="true"/>
```

Comment:
sorry it was my typo , please do `<property name="yourPossibleProperty2" value="some value"/>`

Reply:
done